content/01.abstract.md

```diff
@@ -7,4 +7,5 @@ We collected keynote speaker invitations for the international meetings held by
 We compared these honorees with last and corresponding author contributions in field-specific journals.
 We used multiple methods to estimate the race, ethnicity, gender, and name groupings of authors and the recipients of these honors.
 To address weaknesses in existing approaches, we built a new dataset of more than 700,000 people with name-nationality pairs from Wikipedia and trained long short-term memory neural networks to make predictions.
-Every approach consistently suggests that white scientists are overrepresented among speakers and honorees, while scientists of color are underrepresented.
+Although we observe similar gender distributions between the group of honorees and authors, other biases might contribute to an underestimation of female scientists in the field, and the proportions have not reached parity.
```

Comment:
```suggestion
Although we observe similar gender distributions between the group of honorees and authors, the proportion of female scientists in the field has not yet reached parity.
```
I don't think it's an underestimation (at least, we don't have evidence for that) but we do have evidence for not parity so I left it this way. See what you think.